mongodbatlas/fw_resource_mongodbatlas_alert_configuration.go

```diff
@@ -0,0 +1,811 @@
+	client, err := ConfigureClientInResource(req.ProviderData)
+	if err != nil {
+		resp.Diagnostics.AddError(errorConfigureSummary, err.Error())
+		return
+	}
+	r.client = client
+}
+
+func (r *AlertConfigurationRS) Schema(ctx context.Context, req resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Computed: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"project_id": schema.StringAttribute{
+				Required: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplace(),
+				},
+			},
+			"alert_configuration_id": schema.StringAttribute{
+				Computed: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"event_type": schema.StringAttribute{
+				Required: true,
+			},
+			"created": schema.StringAttribute{
+				Computed: true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.UseStateForUnknown(),
+				},
+			},
+			"updated": schema.StringAttribute{
+				Computed: true,
+			},
+			"enabled": schema.BoolAttribute{
+				Computed: true,
+				Optional: true,
+			},
+		},
+		Blocks: map[string]schema.Block{
+			"matcher": schema.ListNestedBlock{
+				NestedObject: schema.NestedBlockObject{
+					Attributes: map[string]schema.Attribute{
+						"field_name": schema.StringAttribute{
+							Optional: true,
+						},
+						"operator": schema.StringAttribute{
+							Optional: true,
+						},
+						"value": schema.StringAttribute{
+							Optional: true,
+						},
+					},
+				},
+			},
+			"metric_threshold_config": schema.ListNestedBlock{
+				Validators: []validator.List{
+					listvalidator.SizeAtMost(1),
+				},
+				NestedObject: schema.NestedBlockObject{
+					Attributes: map[string]schema.Attribute{
+						"metric_name": schema.StringAttribute{
+							Optional: true,
+						},
+						"operator": schema.StringAttribute{
+							Optional: true,
+							Validators: []validator.String{
+								stringvalidator.OneOf("GREATER_THAN", "LESS_THAN"),
+							},
+						},
+						"threshold": schema.Float64Attribute{
+							Optional: true,
+						},
+						"units": schema.StringAttribute{
+							Optional: true,
+						},
+						"mode": schema.StringAttribute{
+							Optional: true,
+						},
+					},
+				},
+			},
+			"threshold_config": schema.ListNestedBlock{
+				Validators: []validator.List{
+					listvalidator.SizeAtMost(1),
+				},
+				NestedObject: schema.NestedBlockObject{
+					Attributes: map[string]schema.Attribute{
+						"operator": schema.StringAttribute{
+							Optional: true,
+						},
+						"threshold": schema.Float64Attribute{
+							Optional: true,
+						},
+						"units": schema.StringAttribute{
+							Optional: true,
+							Validators: []validator.String{
+								stringvalidator.OneOf(
```

Comment:
Does this validator ignoreCase as in validation.StringInSlice()? See previous [implementation](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/mongodbatlas/resource_mongodbatlas_alert_configuration.go#L155)
Applicable to all other uses here

Reply:
Previous implementation defines `ignoreCase` as false for all StringInSlice calls, meaning it is case sensitive. I have verified that stringvalidator.OneOf() is also caes sensitive, so we should be good here.

Reply:
Great, thank you!